core/src/main/java/cocktailofferer/service/image/command/ImageMetadataCommandService.java

```diff
@@ -0,0 +1,5 @@
+package cocktailofferer.service.image.command;
+
+public interface ImageMetadataCommandService {
+    CreateImageMetadataCommandResult create(CreateImageMetadataCommand command);
```

Comment:
As CreateImageMetadataCommand contains validation rules, bean validation should be enabled somewhere. And a test? ))